### PR TITLE
updated report queue and cost docs

### PR DESCRIPTION
**Pull request by andre8244**
Expand NethVoice report documentation

nethesis/dev#5907

### CODE REVIEW

File: admin-manual/report.rst [hunk 1]
```diff
@@ -27,11 +27,58 @@ Quindi, al termine della prima installazione, è necessario attendere il giorno
...
-Report chiamate & report costi
-==============================
+Grafici
+-------
 
-Ancora **NON** implementato.
-I dati sono consultabili dal :ref:`vecchio report chiamate <inboundreport-ref-label>` e da |product_cti|.
+L'area dei grafici costituisce quella di maggior interesse per l'utente e costituisce il corpo del report della vista corrente.
+Ciascun grafico può essere esportato in almeno uno dei seguenti formati: CSV, PNG e PDF.
+Per motivi di leggibilità alcuni grafici mostrano soltanto i dati più rilevanti: il pulsante :guilabel:`Mostra dettagli` è possibile accedere al set completo dei dati del grafico.
```
Comment (gsanchietti) [inline admin-manual/report.rst:83]:
```suggestion
Per motivi di leggibilità, alcuni grafici mostrano soltanto i dati più rilevanti: attraverso il pulsante :guilabel:`Mostra dettagli` è possibile accedere al set completo dei dati del grafico.
```

**Review by edospadoni — APPROVED** [review]

File: admin-manual/report.rst [hunk 2]
```diff
@@ -27,11 +27,136 @@ Quindi, al termine della prima installazione, è necessario attendere il giorno
...
+
+.. _report-settings-ref-label:
+
+Impostazioni
+============
+
+Le impostazioni dei report sono accessibili cliccando il pulsante con l'icona di ingranaggio nella barra degli strumenti in alto a destra.
+Il pulsante è visibile soltanto se è stato effettuato il login con l'utenza ``admin``.
+
+le impostazioni sono organizzate nelle seguenti sezioni:
```
Comment (gsanchietti) [inline admin-manual/report.rst:101]:
```suggestion
Le impostazioni sono organizzate nelle seguenti sezioni:
```

File: admin-manual/report.rst [hunk 3]
```diff
@@ -27,11 +27,136 @@ Quindi, al termine della prima installazione, è necessario attendere il giorno
...
+
+Esempio di configurazione di un nuovo costo
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Si supponga di avere attivato su un fascio PBX di nome `trunk_1` un contratto telefonico secondo il quale le chiamate verso la Spagna hanno una tariffazione di 0.01 EUR al secondo. Per far sì che il costo di queste chiamate sia calcolato e mostrato nel report, è necessario seguire i seguenti passi:
+
+- Accedere al report con utenza `admin`
+- Accedere alle impostazioni
+- Definire una nuova destinazione, denominandola ad esempio `Spagna`
+- Configurare un nuovo prefisso di destinazione, indicando il prefisso nazionale spagnolo (`0034` oppure `+34`, in funzione di come è stato configurato il centralino) e selezionando `Spagna` come destinazione
```
Comment (gsanchietti) [inline admin-manual/report.rst:149]:
```suggestion
- Configurare un nuovo prefisso di destinazione, indicando il prefisso nazionale spagnolo (`0034` oppure `+34`, in funzione di come è stato configurato il centralino) e selezionare `Spagna` come destinazione
```

**Review by gsanchietti — CHANGES_REQUESTED** [review]

File: admin-manual/report.rst (hunk 3 above)
Comment (gsanchietti) [reply admin-manual/report.rst:149]: Let's keep original form

**Review by gsanchietti — APPROVED** [review]

**Pull request merged**